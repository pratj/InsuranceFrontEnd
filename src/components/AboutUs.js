import React from "react";

function AboutUs() {
  return (
    <div>
      <h1 style={{ textAlign: "center" }}>About Us</h1>
      <marquee width="60%" direction="left" height="100px">Prateek Param Rashwin</marquee>
    </div>
  );
}

export default AboutUs;
