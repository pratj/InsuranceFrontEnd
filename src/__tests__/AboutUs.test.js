import { shallow } from "enzyme";
import AboutUs from "../components/AboutUs";
import { render, screen } from "@testing-library/react";
import toJson from "enzyme-to-json";

test("renders aboutUs component", () => {
  shallow(<AboutUs />);
});

test("renders aboutUs component with parent div", () => {
  const wrapper = shallow(<AboutUs />);
  expect(wrapper.find('[data-test="aboutUs"]').length).toBe(1);
});

test("render names", () => {
  render(<AboutUs />);
  expect(screen.getByText(/prateek param rashwin/i)).toBeInTheDocument();
});

test("AboutUs snapshot", () => {
  const aboutUsTree = shallow(<AboutUs />);
  expect(toJson(aboutUsTree)).toMatchSnapshot();
});