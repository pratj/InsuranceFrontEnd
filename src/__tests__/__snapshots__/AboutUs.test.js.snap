// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs snapshot 1`] = `
<div
  className="aboutUs"
  data-test="aboutUs"
>
  <h1
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    About Us
  </h1>
  <marquee
    direction="left"
    height="100px"
    width="60%"
  >
    Prateek Param Rashwin
  </marquee>
</div>
`;
